documentation/assemblies/configuring/assembly-reassign-tool.adoc

```diff
@@ -16,10 +16,17 @@ The tool can also be used to reassign partitions and balance the distribution of
 However, it is recommended to use xref:cruise-control-concepts-str[Cruise Control for automated partition reassignments and cluster rebalancing]. 
 Cruise Control can move topics from one broker to another without any downtime, and it is the most efficient way to reassign partitions.
 
+NOTE: It is recommended to run the `kafka-reassign-partitions.sh` tool as a separate interactive pod rather than within the broker container.
+Running the Kafka `bin/` scripts within the broker container may cause a JVM to start with the same settings as the Kafka broker, which can potentially cause disruptions.
+By running the `kafka-reassign-partitions.sh` tool in a separate pod, you can avoid this issue.
+Running a pod with the `-ti` option creates an interactive pod with a terminal for running shell commands inside the pod.
```

Comment:
I wonder if we should be a bit more explicit here and add an example:

```
kubectl run helper-pod -ti --image=quay.io/strimzi/kafka:0.33.2-kafka-3.4.0 --rm=true --restart=Never -- bash
```

(and just have a placeholders for the versions)

Reply:
Yep. I've added with the `{DockerKafka}` attribute so it picks up the latest.